h5p-bildetema/src/components/Toggle/ReactToggle.scss

```diff
@@ -0,0 +1,141 @@
+@use "../../../../common/abstracts" as *;
+
+.react-toggle {
+  touch-action: pan-x;
+
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  background-color: transparent;
+  border: 0;
+  padding: 0;
+
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
Do we need all these, or are they added automatically?

Reply:
They are from the original css for the react-toggle, but we can remove them